### PR TITLE
Add user-feedback to go-copy

### LINKED ISSUE #613: [Feature] Add user-feedback to go-copy
### What is the feature that you would like to propose?
We should add some sort of user-feedback to indicate when the text tied to a `go-copy` text has been successfully copied upon clicking it.

### PR DESCRIPTION
**Pull request by jaredami**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](https://github.com/mobi/goponents/blob/master/CONTRIBUTING.md)
- [x] Tests for the changes have been added
- [ ] Docs have been added or updated

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Style Update (CSS)
- [ ] Other... Please describe:

## What is the current behavior?

Resolves #613 

## What is the new behavior?
When a user clicks on a `go-copy`, a green checkmark appears for 3 seconds to indicate that the text was successfully copied.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/go-lib/src/lib/components/go-copy/go-copy.component.scss [hunk 1]
```diff
@@ -22,3 +24,7 @@
   overflow: hidden;
   position: absolute;
 }
+
+.go-copy__icon--clicked {
+  color: $base-primary;
```
Comment (grahamhency) [inline projects/go-lib/src/lib/components/go-copy/go-copy.component.scss:29]: What do you think about using our `$ui-color-positive` variable for this instead?

File: projects/go-lib/src/lib/components/go-copy/go-copy.component.ts [hunk 2]
```diff
@@ -1,7 +1,35 @@
...
+    trigger('checkMarkFadeInOut', [
+      state('content_copy', style({
+        opacity: '1'
+      })),
+      state('check', style({
+        opacity: '1'
+      })),
+      transition('content_copy => check', [
+        group([
+          animate('3s', keyframes([
```
Comment (grahamhency) [inline projects/go-lib/src/lib/components/go-copy/go-copy.component.ts:15]: So I am kind of thinking we can hasten this a bit.. what do you think about cutting this down to `1.5s`?

File: projects/go-lib/src/lib/components/go-copy/go-copy.component.scss (hunk 1 above)
Comment (jaredami) [reply projects/go-lib/src/lib/components/go-copy/go-copy.component.scss:29]: Oh yep, that would make more sense 👍

File: projects/go-lib/src/lib/components/go-copy/go-copy.component.ts (hunk 2 above)
Comment (jaredami) [reply projects/go-lib/src/lib/components/go-copy/go-copy.component.ts:15]: Yeah, coming back to this after awhile, it does feel long. `1.5s` it is 👍

**Pull request merged**